index.html

```diff
@@ -0,0 +1,210 @@
+          <div class="col-md-6 info-box-item">
+            <div class="info-box-title">AUTONOMY FOR ALL</div>
+            <p class="info-box-description">
+              DAVI makes on-chain governance <strong> accessible</strong>,
+              offering a painless and detailed UI that allows
+              <strong>everybody</strong> to get started with a DAO.
+            </p>
+          </div>
+        </article>
+        <article class="modal-body row info-box-article">
+          <img
+            src="assets/black-90.png"
+            alt="black-circles"
+            class="black-90-mobile info-box-item col-md-6"
+          />
+          <div class="col-md-6 info-box-item">
+            <div class="info-box-title">FULLY DECENTRALIZED</div>
+            <p class="info-box-description">
+              DAVI is <strong>decentralized from front to back</strong>. It
+              enables DAOs to vote in other DAOs and includes social
+              integrations that emphaise user ownership.
+              <strong> On-chain always comes first.</strong>
+            </p>
+          </div>
+          <img
+            src="assets/black-90.png"
+            alt="black-circles"
+            class="black-90-desktop info-box-item col-md-6"
+          />
+        </article>
+        <article class="modal-body row info-box-article">
+          <img
+            src="assets/black-7.png"
+            alt="black-7"
+            class="info-box-item col-md-6"
+          />
+          <div class="col-md-6 info-box-item">
+            <div class="info-box-title">YOUR DAO YOUR WAY</div>
+            <p class="info-box-description">
+              DAVI provides <strong> flexible architecture </strong>
+              that adapts to the logic of your DAO. From soulbound guilds, to
+              advanced Governance 2.0 DAO's,
+              <strong>handle it your way</strong> with DAVI.
+            </p>
+          </div>
+        </article>
+      </section>
+    </div>
+    <section class="socials">
+      <div class="socials-section">
+        <a
+          href="#"
+          class="btn-social button btn btn-primary btn-lg"
+          role="button"
+          >TWITTER
+          <img src="assets/arrow.png" alt="arrow" class="arrow" />
+        </a>
+        <a
+          href="#"
+          class="btn-social button btn btn-primary btn-lg"
+          role="button"
+          >DISCORD
+          <img src="assets/arrow.png" alt="arrow" class="arrow" />
+        </a>
+        <a
+          href="#"
+          class="btn-social button btn btn-primary btn-lg"
+          role="button"
+          >GITHUB
+          <img src="assets/arrow.png" alt="arrow" class="arrow" />
+        </a>
+      </div>
+    </section>
+    <section class="links-section">
+      <div class="links-group">
+        <div class="link-title">ABOUT</div>
+        <hr class="divider" />
+        <div>
+          <a href="#"> Faq </a>
+        </div>
+        <div><a href="#"> Blog </a></div>
```

Comment:
Rename to "Updates"
https://discord.com/channels/795724884763738173/1011427091910234132